.travis.yml

```diff
@@ -40,6 +40,11 @@ deploy:
     on:
       branch: master
       jdk: openjdk8
+  - provider: releases
+    api_key: "${GITHUB_WRITE_TOKEN}"
+    on:
+      branch: master
+      jdk: openjdk8
```

Comment:
Don't we also need `on.tags: true`?

Reply:
I've had a look at the other release providers, and they only use these two preconditions. If I understood correctly, Pioneer is being pushed to Maven Central whenever something gets merged into `master`. Is this correct? The GitHub release provider here would follow this pattern, i.e., publish on GitHub when merged into `master`.

Reply:
Shipkit only releases commits to master under certain conditions:

- the commit message doesn't contain `[ci skip-release]`
- all checks (e.g. tests) are successful
- at least one main artifact (that includes `...-source.jar` and `...-javadoc.jar`) has changed

It also tags exactly the commits that fulfill these conditions, so `on.tags: true` should do the trick for us.

Reply:
Allright, I see. Good you double-checked this!

AFAIK one cannot combine `on.branch` and `on.tags` (see https://github.com/travis-ci/travis-ci/issues/7488). Are tags only created for releases? Because then it would be sufficient to check for tags only.

Reply:
Yes, at the moment tags are only created for releases, but relying on that seems a little fragile. I don't get why Travis can't check both and asked over at the issue you linked. Let's see what comes of it.

Reply:
No, wait, let's not see. Instead, let's edit and merge this and maybe come back to it in the future if need be.